services/121-service/.knip.json

```
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/knip@5/schema.json",
```

```suggestion
  "$schema": "node_modules/knip/schema.json",
```

Would this work?
Then it would use the currently-installed-version's schema, right?

(I have used a similar pattern in another angular-app, so I guess it should work here as well..)

That way, we're not relying on a third-party-service AND it doesn't need to update for every version-upgrade...